walletnode/services/artworkregister/task_test.go

```diff
@@ -104,12 +105,15 @@ func TestTaskRun(t *testing.T) {
 	}
 
 	t.Run("group", func(t *testing.T) {
-		//create global tmp image file on current dir
-		artworkFile, err := newTestImageFile()
+		//create global tmp dir for store fake image on current dir
+		tmpDir, err := ioutil.TempDir("", "task_run")
```

Comment:
```suggestion
		tmpDir, err := ioutil.TempDir("", "artwork-*")
```

> The filename is generated by taking pattern and adding a random string to the end. If pattern includes a `*`, the random string replaces the last `*`. Multiple programs calling TempFile simultaneously will not choose the same file. 

This is especially useful with `-race`